dotcom-rendering/src/web/components/Discussion.tsx

```diff
@@ -180,62 +164,29 @@ export const Discussion = ({
 								/>
 							</div>
 						</Hide>
-
-						{beingHydrated && isExpanded && (
```

Comment:
Should we test what happens with expanded behaviour

Reply:
+1 to Olly's suggestion